play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/internal/CliOptions.kt

```diff
@@ -265,6 +269,12 @@ internal interface BootstrapOptions {
                 field = value
             }
 
+        override var downloadSubscriptions = true
```

Comment:
Should `downloadProducts` (line 266 above) default to false? Like does it even make sense to have both?

Reply:
No. We should have both "products" and "subscriptions" to true. "Products" (inappproducts.*) still used for in-app purchases (consumable and non-consumable), and "subscriptions" (monetization.subscriptions.*) used only for subscriptions.

Reply:
Sounds good, thanks for the explanation!